### PR TITLE
Update ASP.NET Core what's new

### PR DESCRIPTION
**Pull request by jamesmontemagno**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/whats-new/dotnet-9/overview.md](https://github.com/dotnet/docs/blob/e3746f38f0f8e0e9cb24fffa1720370d51b291ae/docs/core/whats-new/dotnet-9/overview.md) | [What's new in .NET 9](https://review.learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/overview?branch=pr-en-us-43425) |

### CODE REVIEW

File: docs/core/whats-new/dotnet-9/overview.md [hunk 1]
```diff
@@ -63,7 +63,15 @@ ML.NET is an open-source, cross-platform framework that enables integration of c
...
+ASP.NET Core apps built with .NET 9 are secure by default, have expanded support for ahead-of-time complication, improved monitoring and tracing, and with the performance improvements built in you will see higher throughput, faster startup time, and all with less memory usage. ASP.NET Core in .NET 9 includes improvements across:
+
+- Optimized handling of static files, like JavaScript and CSS, at build & publish time with automatic fingerprinted verisioning.
+- Blazor: New Blazor Hybrid and Web app templates, detection of component render mode, new reconnection experience with server rendering.
+- APIs: Built in support for OpenAPI document generation using `Microsoft.AspNetCore.OpenAPI`, enhanced native AOT support,
+- Improved security with new APIs for authentication and authorization.
+- Easier setup for trusted development certificate on Linux to enable HTTPS during development.
+
+These are just some of the features and enhancements you will find in .NET 9, for more information, see [What's new in ASP.NET Core 9.0](/aspnet/core/release-notes/aspnetcore-9.0).
```
Comment (gewarren) [inline docs/core/whats-new/dotnet-9/overview.md:74]:
```suggestion
These are just some of the features and enhancements in .NET 9. For more information, see [What's new in ASP.NET Core 9.0](/aspnet/core/release-notes/aspnetcore-9.0).
```

**Review by danroth27 — APPROVED** [review]
Looks good to me. Thanks @jamesmontemagno!

File: docs/core/whats-new/dotnet-9/overview.md [hunk 2]
```diff
@@ -63,7 +63,15 @@ ML.NET is an open-source, cross-platform framework that enables integration of c
 
 ## ASP.NET Core
 
-ASP.NET Core includes improvements to Blazor, SignalR, minimal APIs, OpenAPI, and authentication and authorization. For more information, see [What's new in ASP.NET Core 9.0](/aspnet/core/release-notes/aspnetcore-9.0).
+ASP.NET Core apps built with .NET 9 are secure by default, have expanded support for ahead-of-time complication, improved monitoring and tracing, and with the performance improvements built in you will see higher throughput, faster startup time, and all with less memory usage. ASP.NET Core in .NET 9 includes improvements across:
+
+- Optimized handling of static files, like JavaScript and CSS, at build & publish time with automatic fingerprinted verisioning.
+- Blazor: New Blazor Hybrid and Web app templates, detection of component render mode, new reconnection experience with server rendering.
+- APIs: Built in support for OpenAPI document generation using `Microsoft.AspNetCore.OpenAPI`, enhanced native AOT support,
```
Comment (gewarren) [inline docs/core/whats-new/dotnet-9/overview.md:70]:
```suggestion
- APIs: Built in support for OpenAPI document generation using `Microsoft.AspNetCore.OpenAPI`, enhanced native AOT support.
```

File: docs/core/whats-new/dotnet-9/overview.md [hunk 3]
```diff
@@ -63,7 +63,15 @@ ML.NET is an open-source, cross-platform framework that enables integration of c
 
 ## ASP.NET Core
 
-ASP.NET Core includes improvements to Blazor, SignalR, minimal APIs, OpenAPI, and authentication and authorization. For more information, see [What's new in ASP.NET Core 9.0](/aspnet/core/release-notes/aspnetcore-9.0).
+ASP.NET Core apps built with .NET 9 are secure by default, have expanded support for ahead-of-time complication, improved monitoring and tracing, and with the performance improvements built in you will see higher throughput, faster startup time, and all with less memory usage. ASP.NET Core in .NET 9 includes improvements across:
+
+- Optimized handling of static files, like JavaScript and CSS, at build & publish time with automatic fingerprinted verisioning.
+- Blazor: New Blazor Hybrid and Web app templates, detection of component render mode, new reconnection experience with server rendering.
```
Comment (gewarren) [inline docs/core/whats-new/dotnet-9/overview.md:69]:
```suggestion
- Blazor: New Hybrid and Web app templates, detection of component render mode, new reconnection experience with server rendering.
```

File: docs/core/whats-new/dotnet-9/overview.md [hunk 4]
```diff
@@ -63,7 +63,15 @@ ML.NET is an open-source, cross-platform framework that enables integration of c
 
 ## ASP.NET Core
 
-ASP.NET Core includes improvements to Blazor, SignalR, minimal APIs, OpenAPI, and authentication and authorization. For more information, see [What's new in ASP.NET Core 9.0](/aspnet/core/release-notes/aspnetcore-9.0).
+ASP.NET Core apps built with .NET 9 are secure by default, have expanded support for ahead-of-time complication, improved monitoring and tracing, and with the performance improvements built in you will see higher throughput, faster startup time, and all with less memory usage. ASP.NET Core in .NET 9 includes improvements across:
+
+- Optimized handling of static files, like JavaScript and CSS, at build & publish time with automatic fingerprinted verisioning.
```
Comment (gewarren) [inline docs/core/whats-new/dotnet-9/overview.md:68]:
```suggestion
- Optimized handling of static files, like JavaScript and CSS, at build and publish time with automatic fingerprinted versioning.
```

File: docs/core/whats-new/dotnet-9/overview.md [hunk 5]
```diff
@@ -63,7 +63,15 @@ ML.NET is an open-source, cross-platform framework that enables integration of c
 
 ## ASP.NET Core
 
-ASP.NET Core includes improvements to Blazor, SignalR, minimal APIs, OpenAPI, and authentication and authorization. For more information, see [What's new in ASP.NET Core 9.0](/aspnet/core/release-notes/aspnetcore-9.0).
+ASP.NET Core apps built with .NET 9 are secure by default, have expanded support for ahead-of-time complication, improved monitoring and tracing, and with the performance improvements built in you will see higher throughput, faster startup time, and all with less memory usage. ASP.NET Core in .NET 9 includes improvements across:
```
Comment (gewarren) [inline docs/core/whats-new/dotnet-9/overview.md:66]:
```suggestion
ASP.NET Core apps built with .NET 9 are secure by default, have expanded support for ahead-of-time complication, and have improved monitoring and tracing. With the performance improvements, you'll see higher throughput and faster startup time, and all with less memory usage. ASP.NET Core in .NET 9 includes:
```

**Review by gewarren — APPROVED** [review]
Left some mostly grammatical suggestions.

**Pull request merged**